pkg/visor/os_executer.go

```diff
@@ -0,0 +1,49 @@
+package visor
```

Comment:
Sorry for the change of plan, but it might make sense to put this into `appserver` (or something else that we decide on).

Reply:
@evanlinjin well, if we encapsulate all app controlling stuff within `appserver`, than yes. I'll do that

Reply:
@Darkren yes I think that will be a good idea.

Reply:
Done partially, still needs refactoring

Reply:
@Darkren is it complete now?